docker/README.rst

```diff
@@ -3,7 +3,7 @@ Docker Deployment
 
 .. TODO
 
-    Docs to update:
+    Is there some way to simplify these instructions? It's still a long, error-prone process! Do we need another setup script that follows these directions and prompts the user for input (do you want to run the server or do development? What is your github userid? Etc.)? Such a script would be able to take care of the first 7 steps, which would be good progress. But, we'd need os-dependent stuff for the first bits (install WSL/Ubuntu on Windows); can we assume Python 3 on Mac?
```

Comment:
I like the idea of a script that can automate even more of this.
  It would allow us to autodetect if they are on windows or linux or whatever.

I wonder if there isn't some kind of command we can have them do at the end of each step to ensure that the step was successful.

Reply:
There is not much we can do other than remind people that Docker on macOS and Windows is not as rock solid as it is on Linux.  Sometimes you just have to reboot.  Its not the fault of Runestone its a flaw in Docker.  -- Which for me has been incredibly stable for the last several weeks...  They have come a long way in get docker stable on the Apple Silicon.  I guess the advice should be that if docker run hello-world does not work then you need to restart the docker desktop.

Reply:
Resolving for now -- these are longer-term goals than just this PR.